pupa/tests/importers/test_membership_importer.py

```diff
@@ -28,14 +28,15 @@ def test_full_membership():
     robot = Person.objects.create(id="robot", name="R. Daneel Olivaw")
     post = Post.objects.create(id='f', label="founder", role="Founder", organization=org)
 
-    # add a membership through a post
-    m1 = ScrapeMembership(person_id=hari.id, organization_id=org.id, post_id=post.id)
+    # add a membership through a post, with a start date
+    m1 = ScrapeMembership(person_id=hari.id, organization_id=org.id,
```

Comment:
do we want a test where there is already and end data one the model, but we ignore that for resolution purposes if there's a start date? that seems like the case we are trying to handle?

Reply:
Good call. Updated the `m1` test case in https://github.com/opencivicdata/pupa/pull/336/commits/045288d737a49a764152112626c8313703c50b0a to resemble this situation.